src/features.xml

```diff
@@ -179,4 +179,11 @@
         </alwaysEnabledBrandingTokens>
     </feature>
 
+    <feature>
+        <name>Feature_DebugMode</name>
+        <description>Enables the debug mode setting</description>

```

Comment:
```suggestion
        <description>Enables UI access to the debug mode setting</description>
```
	
right?

Reply:
the feature name makes it sound quite ominous

Reply:
Applied. Also updated name to `Feature_DebugModeUI`

Reply:
much appreciated